thirdeye-server/src/main/java/ai/startree/thirdeye/rca/RcaInfoFetcher.java

```diff
@@ -131,20 +131,10 @@ public RcaInfo getRcaInfo(final AnomalyDTO anomalyDTO)
     // we get the namespace from the alert and inject it in all persistence access - authz is not performed - the consumer of this method should ensure authz on the anomaly
     final String alertNamespace = alertDTO.namespace();
     MetricConfigDTO metricConfigDTO = metricDAO.findBy(metricName, datasetName, alertNamespace);
```

Comment:
I think we should not fetch the metric config from the persistence layer anymore and construct it manually.

Will put more details in the other PR